addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -40,7 +40,6 @@
         <field name="model">hr.leave</field>
         <field name="arch" type="xml">
             <search string="Search Time Off">
-                <field name="employee_id"/>
```

Comment:
This will not do it; that will indeed remove the filter on the dashboard, but it will also remove it in the management filters. This should be made invisible in `hr_leave_view_search_my` instead
```xml
            <xpath expr="//filter[@name='employee_id']" position="attributes">
                <attribute name="invisible">1</attribute>
            </xpath>
```